core/cas-server-core-services/src/main/java/org/apereo/cas/config/CasCoreServicesConfiguration.java

```diff
@@ -140,6 +142,13 @@ public RegisteredServiceReplicationStrategy registeredServiceReplicationStrategy
         return new NoOpRegisteredServiceReplicationStrategy();
     }
 
+    @ConditionalOnMissingBean(name = "registeredServiceresourceNamingStrategy")
+    @Bean
+    @RefreshScope
+    public RegisteredServiceResourceNamingStrategy resourceNamingStrategy() {
```

Comment:
`registeredServiceResourceNamingStrategy`, and the actual method name should change too.